jobs/build/signed-compose/build.groovy

```diff
@@ -22,12 +22,21 @@ def initialize(advisory) {
 
 // Set the advisory to the NEW_FILES state (allows builds to be added)
 def signedComposeStateNewFiles() {
+    if (params.DRY_RUN) {
+        echo("Skipping advisory state change for dry run.")
+        echo("would have run: ${elliottOpts} change-state --state NEW_FILES ${advisoryOpt}")
+        return
+    }
     buildlib.elliott("${elliottOpts} change-state --state NEW_FILES ${advisoryOpt}")
 }
 
 // Search brew for, and then attach, any viable builds. Do this for
 // EL7 and EL8.
 def signedComposeAttachBuilds() {
+    if (!params.ATTACH_BUILDS) {
+        echo "Job configured not to attach builds; continuing using builds already attached"
```

Comment:
tiny thing, but (almost) all the other `echo`s use `echo(msg)` (looks like I missed `Action may be required`).

Reply:
i don't mind changing it, but maybe we should establish a jenkins pipeline style guide :-P